tmt/steps/provision/local.py

```diff
@@ -0,0 +1,79 @@
+import re
+import tmt
+
+class ProvisionLocal(tmt.steps.provision.ProvisionPlugin):
+    """
+    Use local host for test execution
+
+    In general it is not recommended to run tests on your local machine
+    as there might be security risks. Run only those tests which you
+    know are safe so that you don't destroy your laptop ;-)
+
+    Example config:
+
+        provision:
+            how: local
+
+    Note that 'tmt run' is expected to be executed under a regular user.
+    If there are admin rights required (for example in the prepare step)
+    you might be asked for a sudo password.
+    """
+
+    # Guest instance
+    _guest = None
+
+    # Supported methods
+    _methods = [tmt.steps.Method(name='local', doc=__doc__, order=50)]
+
+    def wake(self, data=None):
+        """ Override options and wake up the guest """
+        if data:
+            self._guest = GuestLocal(data, name=self.name, parent=self.step)
+
+    def go(self):
+        """ Provision the container """
+        super().go()
+
+        # Give a short info about localhost
```

Comment:
hmm, seems something useful in general, not just localhost ....

Reply:
Nice idea! Moved the functionality to the `Guest` class method `details()`.

Reply:
Example output from the `container` provisioner:

```
/plans/smoke
    discover
        how: shell
        directory: /home/psss/git/tmt
        summary: 1 test selected
    provision
        how: container
        image: fedora:fresh
        distro: Fedora release 31 (Thirty One)
        kernel: 5.5.11-200.fc31.x86_64
        summary: 1 guest provisioned
```